assets/sass/custom.sass

```diff
@@ -459,7 +459,7 @@ a.top-banner
       max-width: 100%
 
     .banner-logo
-      background-image: url('/images/events/o3dcon_2022_1.png')
+      background-image: url('/images/events/o3d_monochrome_con_logo.svg')
```

Comment:
Minor: could you add `margin-top: -2px` here? That way the logo would look centered:
![image](https://user-images.githubusercontent.com/82231674/180050797-f38cc113-bbe7-4716-a4cf-ce269c9b62b5.png)

Reply:
Yes, Will do that now, :)